autoload/fzf/vim.vim

```diff
@@ -1210,6 +1210,38 @@ function! fzf#vim#jumps(...)
   \ }, a:000)
 endfunction
 
+" ------------------------------------------------------------------
+" Changes
+" ------------------------------------------------------------------
+function! s:format_change(line)
+  return substitute(a:line, '\d\{1,2}', '\=s:yellow(submatch(0), "Number")', '')
+endfunction
+
+function! s:change_sink(lines)
+  if len(a:lines) < 2
+    return
+  endif
+  let cmd = s:action_for(a:lines[0])
+  if !empty(cmd)
+    execute 'silent' cmd
+  endif
+  let linecol = split(a:lines[1])
+  execute 'normal! '.linecol[1].'Gzz'
+endfunction
+
+function! fzf#vim#changes(...)
+  redir => cout
+  silent changes
+  redir END
+  let s:changelist = split(cout, "\n")
+  let current = -match(s:changelist, '\v^\s*\>')
+  return s:fzf('changes', {
+  \ 'source':  extend(s:changelist[0:0], map(s:changelist[1:], 's:format_change(v:val)')),
```

Comment:
Don't have to be an `s:` variable.

```suggestion
  let list = split(cout, "\n")
  let current = -match(list, '\v^\s*\>')
  return s:fzf('changes', {
  \ 'source':  extend(list[0:0], map(list[1:], 's:format_change(v:val)')),
```

Reply:
https///github.com/junegunn/fzf.vim/pull/1478/commits/b90f4d3